torch/onnx/symbolic_opset9.py

```diff
@@ -1493,6 +1493,19 @@ def scalar_tensor(g, scalar, dtype, *options):
     scalar = g.op("Cast", scalar, to_i=sym_help.scalar_type_to_onnx[dtype])
     return scalar
 
+def tensor(g, data, dtype=None, device=None, requires_grad=False):
+    dtype = sym_help._get_const(dtype, 'i', 'dtype')
+    if dtype is None:
+        dtype = 4  # int64
```

Comment:
I don't think it's a good idea to hardcode dtype to int64. The torch op aten::tensor can have a float input, in that case output will also be float if dtype is not specified.
Repro would be:
```
        class FloatInputModel(torch.jit.ScriptModule):
            @torch.jit.script_method
            def forward(self, input):
                return torch.tensor([float(input)])

        x = torch.randn(1)
        self.run_test(FloatInputModel(), x)
```
You can check whether dtype is a None = prim:Const node, and cast only in that case.

```
    if dtype and dtype.node().kind() != 'prim::Constant':
        data_type = sym_help._get_const(dtype, 'i', 'dtype')
        ... cast ...
```